apps/indexer/lib/indexer/fetcher/empty_blocks_sanitizer.ex

```diff
@@ -80,53 +83,97 @@ defmodule Indexer.Fetcher.EmptyBlocksSanitizer do
       set: [is_empty: false, updated_at: Timex.now()]
     )
 
-    unprocessed_empty_blocks_from_db = unprocessed_empty_blocks_query_list(limit())
-
-    unprocessed_empty_blocks_from_db
-    |> Enum.with_index()
-    |> Enum.each(fn {{block_number, block_hash}, ind} ->
-      with {:ok, %{"transactions" => transactions}} <-
-             %{id: ind, method: "eth_getBlockByNumber", params: [integer_to_quantity(block_number), false]}
-             |> request()
-             |> json_rpc(json_rpc_named_arguments) do
-        transactions_count =
-          transactions
-          |> Enum.count()
-
-        if transactions_count > 0 do
-          Logger.info(
-            "Block with number #{block_number} and hash #{to_string(block_hash)} is full of transactions. We should set consensus = false for it in order to refetch.",
-            fetcher: :empty_blocks_to_refetch
-          )
+    unprocessed_empty_blocks_list =
+      limit()
+      |> unprocessed_empty_blocks_list_query
```

Comment:
```suggestion
    unprocessed_empty_blocks_list = unprocessed_empty_blocks_list_query(limit())
```

According to style guide it's [recommended to not use pipe operator just once](https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#avoid-single-pipelines) and [always use parentheses in function calls](https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#parentheses)